startups/lib/main_0.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:english_words/english_words.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Namer App',
+      theme: ThemeData(
+        useMaterial3: true,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepOrange),
+      ),
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  var current = WordPair.random();
```

Comment:
We're already starting with a 'business logic' because that's the last thing I want people to think about.

Reply:
Hmm. I'd be tempted to mention turning this var private and pairing it with a public `get` method to show case that Dart can be used to do implementation hiding.

Reply:
I like this. I think the best approach here is to point this out and maybe show how the code would look like (if we have time, of course), but not to actually implement it. Since privacy in Dart is at library (~file) level, not class level, students could be confused that the implementation hiding doesn't work.